src/components/plugin/pane.js

```diff
@@ -0,0 +1,147 @@
+'use strict'
+
+const React = require('react')
+const { Component } = React
+const { ipcRenderer: ipc } = require('electron')
+const { PrefPane } = require('../prefs/pane')
+const { Button } = require('../button')
+const { bool, func, object, string } = require('prop-types')
+const { AccordionGroup } = require('../accordion')
+const { PluginAccordion } = require('./accordion')
+const { keys, values } = Object
+
+
+class PluginsPane extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      config: props.plugins.config,
+      spec: props.plugins.spec
+    }
+  }
+
+  componentDidMount() {
+    this.props.plugins.on('change', this.refresh)
+  }
+
+  componentWillUnmount() {
+    this.props.plugins.removeListener('change', this.refresh)
+  }
+
+  componentWillReceiveProps(props) {
+    this.refresh(props)
+  }
+
+  refresh = (props = this.props) => {
+    this.setState({
+      spec: { ...props.plugins.spec },
+      config: [...props.plugins.config],
+    })
+  }
+
+  onChange = (plugin, index, newConfig) => {
```

Comment:
These methods, they shouldn't be called `onSomething`, but rather `handleSomething` - to respect the convention, right?

Reply:
Yes